global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "8.0.302"
+    "version": "8.0.403"
```

Comment:
reverted

Reply:
Decided to update to 8.0.401 because if we keep using 8.0.302 then we have error build if we install 8.0.403 then revert to 8.0.302. 8.0.401 working fine